src/Symfony/Component/Mailer/Tests/Transport/FailoverTransportTest.php

```diff
@@ -85,16 +85,30 @@ public function testSendOneDead()
     public function testSendOneDeadAndRecoveryWithinRetryPeriod()
     {
         $t1 = $this->createMock(TransportInterface::class);
-        $t1->method('send')->willReturnOnConsecutiveCalls($this->throwException(new TransportException()));
+
+        $t1Matcher = $this->any();
+        $t1->expects($t1Matcher)
+            ->method('send')
+            ->willReturnCallback(function () use ($t1Matcher) {
+                if (1 === $t1Matcher->getInvocationCount()) {
+                    throw new TransportException();
+                }
+
+                return null;
```

Comment:
`send()` expects an `Envelope` to be returned (same everywhere below)

Reply:
The current behavior is that it returns `null` after the first call (because `willReturnOnConsecutiveCalls` is only call with one argument, all other calls will return null). Also, `TransportInterface` allows to return `null` when calling `send`. Should we really make this change?

Reply:
My bad, I looked at the wrong interface. 🙈